src/mem/alloc.rs

```diff
@@ -0,0 +1,1063 @@
+/* SPDX-License-Identifier: MIT */
+/*
+ * Copyright (c) 2022 SUSE LLC
+ *
+ * Author: Joerg Roedel <jroedel@suse.de>
+ *
+ * vim: ts=4 sw=4 et
```

Comment:
Remove this line with Vim config options

Reply:
Done.